### DISCUSSION TITLE
Meeting 3: March 21st 15:00 UTC

### DISCUSSION THREAD

**Discussion by ml-evs** [marda-alliance/metadata_extractors#15] [Announcements]
[21st March 15:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230321T15) on Jitsi at https://meet.jit.si/marda_extractors

~Hi everyone, rather than sending everyone a survey for times in March, those that are highly involved can fill it in here: http://whenisgood.net/d7tr52d The link is also on the MaRDA Slack (let me know if you want an invite).~

~I'll update this posting in a few weeks once we have a date set.~

**Comment by ml-evs** [comment]
Now announced to be [21st March 15:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230321T15)

**Reply by ml-evs** [reply]
Apologies, I sent out an email that said 20th alongside a calendar invite for the 21st, 21st is the correct date!

**Comment by PeterKraus** [comment]
The minutes for the above meeting are now available [here](https://github.com/marda-alliance/metadata_extractors/blob/main/meetings/3-meeting-2023-03-21/minutes.md).
